projects/packages/paypal-payments/src/block/edit.js

```diff
@@ -0,0 +1,588 @@
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import {
+	Disabled,
+	ExternalLink,
+	SelectControl,
+	TextareaControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+import { compose, withInstanceId } from '@wordpress/compose';
+import { dispatch, withSelect } from '@wordpress/data';
+import { useEffect, useRef, useState } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { getWidgetIdFromBlock } from '@wordpress/widgets';
+import clsx from 'clsx';
+import emailValidator from 'email-validator';
+import { get, isEmpty, pick, trimEnd } from 'lodash';
+import { getCurrencyDefaults } from '../../shared/currencies';
+import HelpMessage from '../../shared/help-message';
```

Comment:
You'll need to bring those as well.

Reply:
I don't know if I did this how you would do it so please let me know. I copied the `getCurrencyDefaults` function to the `utils.js` file in the Simple Payments block dir. I then copied the `help-message` JS and SCSS files to the package within the `src/block` directory. I made necessary updates to the `edit.js` file and added the `@automattic/format-currency` dependency. All this was done in [9e281ea](https://github.com/Automattic/jetpack/pull/43413/commits/9e281ea750dccb6b0f1bb46ca92d05d458428e8e).

Reply:
The updates to the `edit.js` file can be found in this commit: [d874ddb](https://github.com/Automattic/jetpack/pull/43413/commits/d874ddbb9a15ddee0b953da86558b750b505e114).

Reply:
I believe that means you'll also need to bring in gridicons as a dependency.

Reply:
🤦  Yep, forgot about that. Fixed in [00bc3d9](https://github.com/Automattic/jetpack/pull/43413/commits/00bc3d936b58e8c79d4c79c8367988b6c6f86ee4).